libraries/chain/worker_evaluators.cpp

```diff
@@ -123,7 +127,7 @@ namespace golos { namespace chain {
         const auto& post = _db.get_comment(o.author, o.permlink);
         const auto& wto = _db.get_worker_techspec(post.id);
 
-        GOLOS_CHECK_LOGIC(wto.state < worker_techspec_state::payment,
+        GOLOS_CHECK_LOGIC(wto.state == worker_techspec_state::payment || wto.state == worker_techspec_state::payment_complete,
```

Comment:
What about approves linked to techspec?
Why don't you check that they are exists.